packages/manager/modules/pci/src/projects/project/storages/databases/database/current-queries/current-queries.constants.js

```diff
@@ -0,0 +1,38 @@
+const TRACKING_PREFIX =
```

Comment:
We already have `trackDashboard` with prefix 
`PublicCloud::pci::projects::project::storages::databases::dashboard::${database.engine}::`
This can be reused for prefix part at least to make sure we can keep prefix in one place.
https://github.com/ovh/manager/blob/99e6f7aee99b1b487b410f40ed1af57d627c7c2a/packages/manager/modules/pci/src/projects/project/storages/databases/database/dashboard.routing.js#L35

Reply:
I tried `trackDashboard` but according to the requirement, `database.engine` is not present in the tracking labels.